agents/eaton_ssh/README.md

```diff
@@ -0,0 +1,5 @@
+# fence_eaton_ssh
+
+This was tested with:
+
```

Comment:
We usually have this info as a comment at the top of the agent (see fence_apc and other agents). I guess you could also mention it in the longdesc of the agent.

Reply:
Thanks. I will correct this with the next push.